src/explore-education-statistics-frontend/src/modules/table-tool/components/TableToolShare.tsx

```diff
@@ -34,17 +38,26 @@ const TableToolShare = ({ tableHeaders, query }: Props) => {
     }
     setPermalinkLoading(true);
 
```

Comment:
We might also want to unset the permalink error here so that it gets removed if we try to generate the link again